### PR TITLE
add note    **请注意这里的lib指的是flink-cdc的lib，而不是flink的lib，pipeline相关的包不要放到flink的lib下，否则会引发类加载相关问题**

### PR DESCRIPTION
**Pull request by Lordeath**
**请注意这里的lib指的是flink-cdc的lib，而不是flink的lib，pipeline相关的包不要放到flink的lib下，否则会引发类加载相关问题**

   **Please note that you need to move the jar to the lib directory of Flink CDC Home, not to the lib directory of Flink Home.**

### CODE REVIEW

**Review by Lordeath — COMMENTED** [review]
self reviewed

File: docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md [hunk 1]
```diff
@@ -186,6 +186,7 @@ MacOS 由于内部实现容器的方式不同，在部署时宿主机直接修
 
 2. 下载下面列出的 connector 包，并且移动到 `lib` 目录下；
    **下载链接只对已发布的版本有效, SNAPSHOT 版本需要本地基于 master 或 release- 分支编译.**
+   **请注意这里的lib指的是flink-cdc的lib，而不是flink的lib，pipeline相关的包不要放到flink的lib下，否则会引发类加载相关问题**
```
Comment (yuxiqian) [inline docs/content.zh/docs/get-started/quickstart/mysql-to-doris.md:189]:
```suggestion
   **请注意，您需要将 jar 移动到 Flink CDC Home 的 lib 目录，而非 Flink Home 的 lib 目录下。**
```

File: docs/content.zh/docs/get-started/quickstart/mysql-to-starrocks.md [hunk 2]
```diff
@@ -147,6 +147,7 @@ under the License.
 
 2. 下载下面列出的 connector 包，并且移动到 lib 目录下；
    **下载链接只对已发布的版本有效, SNAPSHOT 版本需要本地基于 master 或 release- 分支编译。**
+   **请注意这里的lib指的是flink-cdc的lib，而不是flink的lib，pipeline相关的包不要放到flink的lib下，否则会引发类加载相关问题**
```
Comment (yuxiqian) [inline docs/content.zh/docs/get-started/quickstart/mysql-to-starrocks.md:150]:
```suggestion
   **请注意，您需要将 jar 移动到 Flink CDC Home 的 lib 目录，而非 Flink Home 的 lib 目录下。**
```

**Review by yuxiqian — COMMENTED** [review]
Thanks for @Lordeath's quick fix, just left some minor comments about the phrasing.

**Review by yuxiqian — APPROVED** [review]
Thanks for @Lordeath and @leonardBang's great work!

**Review by leonardBang — APPROVED** [review]
Thanks @Lordeath for the contribution and @yuxiqian for the review.

**Pull request merged**